.travis.yml

```diff
@@ -205,3 +205,11 @@ matrix:
               STATUSM="$STATUSM ($(python -c "print '%+.2f' % (100*($CURR-$PREV)/$PREV.0)")%)"
           fi
         - bash -c "$STATUS" success "$STATUSM"
+
+    - env:
+        - NAME=gitattributestest
+      script:
+        # Print git diff to make changes visible
+        - git diff
```

Comment:
If you do actually want to see the output, as well as abort the test, then I suggest using the single command

    git diff --exit-code